### DISCUSSION TITLE
Clip Application on Raspberry Pi with AI Kit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1856] [Q&A]
# Clip Application on Raspberry Pi with AI Kit | Seeed Studio Wiki

This wiki demonstrates how to use clip on raspberry pi5 with AI kit.

https://wiki.seeedstudio.com/clip_application_on_rpi5_with_ai_kit/

**Comment by dje666** [comment]
Are there any missing steps in this tutorial? 

$ python3 -m pip install -v -e .

This fails with error: externally-managed-environment......

I create and activate a new virtual environment, but the get warnings about not being in the Hailo virtual environment....?

A few more instructions would be appreciated.

TTFN,

DJE666

**Reply by dje666** [reply]
Ignore this post. Sent to the wrong topic

**Reply by dje666** [reply]
This issue is regarding the CLIP application on RPi5 and H8.
